.env-sample

```diff
@@ -7,8 +7,7 @@ CRDS_CMS_SERVER_ENDPOINT=
 PUSHPAY_GIVE_ENDPOINT=
 IMGIX_SRC=
 IMGIX_DOMAIN=
-STREAMSPOT_API_KEY
-STREAMSPOT_ID=
+STREAM_SCHEDULE_ENDPOINT=https://8k97vbzbrk.execute-api.us-east-1.amazonaws.com/int/streamSchedule
```

Comment:
do we want the actual value stored in github?

Reply:
maybe not. I'll remove